unity-renderer/Assets/Desktop/Scripts/MainScripts/DCL/MainDesktop/MainDesktop.cs

```diff
@@ -68,7 +68,9 @@ protected override void InitializeCommunication()
 
 #if UNITY_EDITOR
                 withSSL = DebugConfigComponent.i.webSocketSSL;
-                startPort = 5000;
+                startPort = 7666;
```

Comment:
I like the change, `5000` is too common, is this used somewhere else by default where we should change it?

Reply:
We have already decided to use 7666 instead of 5000; this was missing. The Launcher is set to 7666, so this change has no production change. It's mainly for easier profiling and debugging.

More details about the use: https://github.com/decentraland/unity-renderer/issues/1807